chia/daemon/server.py

```diff
@@ -1102,30 +1102,40 @@ async def stop_service(self, request: Dict[str, Any]) -> Dict[str, Any]:
         return response
 
     async def is_running(self, request: Dict[str, Any]) -> Dict[str, Any]:
-        service_name = request["service"]
-
-        if service_name == service_plotter:
-            processes = self.services.get(service_name)
-            is_running = processes is not None and len(processes) > 0
-            response = {
-                "success": True,
-                "service_name": service_name,
-                "is_running": is_running,
-            }
+        service = request.get("service", "*")
+        services_to_check = (
+            list(map(str.strip, service.split(",")))
+            if service != "*"
+            else list(set([*self.services.keys(), *self.connections.keys()]))
+        )
+        responses = []
+        for service_name in services_to_check:
+            if service_name == service_plotter:
+                processes = self.services.get(service_name)
+                is_running = processes is not None and len(processes) > 0
+                response = {
+                    "service_name": service_name,
+                    "is_running": is_running,
+                }
+            else:
+                process = self.services.get(service_name)
+                is_running = process is not None and process.poll() is None
+                if not is_running:
+                    # Check if we have a connection to the requested service. This might be the
+                    # case if the service was started manually (i.e. not started by the daemon).
+                    service_connections = self.connections.get(service_name)
+                    if service_connections is not None:
+                        is_running = len(service_connections) > 0
+                response = {
+                    "service_name": service_name,
+                    "is_running": is_running,
+                }
+            responses.append(response)
+
+        if len(responses) == 1:
+            response = {"success": True, **responses[0]}
         else:
-            process = self.services.get(service_name)
-            is_running = process is not None and process.poll() is None
-            if not is_running:
-                # Check if we have a connection to the requested service. This might be the
-                # case if the service was started manually (i.e. not started by the daemon).
-                service_connections = self.connections.get(service_name)
-                if service_connections is not None:
-                    is_running = len(service_connections) > 0
-            response = {
-                "success": True,
-                "service_name": service_name,
-                "is_running": is_running,
-            }
+            response = {"success": True, "services": responses}
```

Comment:
I guess there's some argument for this in terms of backwards compatibility.  Maybe though we could define a consistent form?  Put the 'first' result into the root object as before but also always provide the `services` attribute?  Then callers don't need to special case one result.  They can just always use the new form from now on.

Reply:
No longer applicable. `is_running` behaves as it did before.